dbaas/physical/models.py

```diff
@@ -513,6 +513,14 @@ def used(self):
         return self.databases.count()
 
     @property
+    def has_custom_parameter(self):
+        parameters = DatabaseInfraParameter.objects.filter(databaseinfra=self)
```

Comment:
you can improve this line
return DatabaseInfraParameter.objects.filter(databaseinfra=self).exists()